src/services/getUniqueItems.js

```diff
@@ -0,0 +1,13 @@
+export function getUniqueItems(items) {
```

Comment:
Same with the items. Why is this called unique and not just Items?

Reply:
because i have to sort out all double items here